### ISSUE TITLE
Review and validation on the boundary data and missing/incorrect street names within the City of Dasmarinas

### ISSUE THREAD

**Issue by DP24PH** [OSMPH/papercut_fix#76]
# INTRODUCTION

Upon the receipt of one of @DP24PH OSM DM from @dasmarinas-mapper regarding the legality of the boundary for the City of Dasmarinas, there are several places to be surveyed for review on first checkmark (based on both the ground and [RA 9723](https://lawphil.net/statutes/repacts/ra2009/ra_9723_2009.html)) and validation for the second checkmark:

- [x] Avida Settings Phase 1 (Bacoor-Dasmarinas boundary) - _validation required for northeastern boundary_
> - [x] collect latest Mapillary image within Bacoor-Dasmarinas National Road - _taken care and courtesy of Kaart_
- [x] Amore Phase 2 and 3 (Bacoor-Dasmarinas-NCR regional boundary) - _validation required for eastern boundary_
- [x] Extreme Southern part of Golden City, from south of Iodine Street (Dasmarinas-Imus boundary) - _validation required for northwestern boundary_
> - [ ] collect latest Mapillary image within J.T. Paterno Boulevard
- [x] Langkaan 2 (Humayao) (Dasmarinas-General Trias-Silang boundary) - _validation required for southwestern boundary_
- [x] Northern part of the International Institute of Rural Reconstruction (Dasmarinas-Silang boundary) - _address involving city or municipality got removed for the meantime, retaining the highway name_
> - [x] collect latest Mapillary image within PH-410 and PH-419/R-2/General Emilio Aguinaldo Highway
- [x] Saint Joseph Ridgeview (Dasmarinas-San Pedro boundary) - _confirmed by P.A. Properties source, with indicated area of uncertainty for the inter-city boundary_

However, one of the review for the **Bacoor-Dasmarinas boundary** within the **`[Bacoor-Dasmarinas] National Road`**, `Molino Road` segment shows that the [welcome arch](https://www.openstreetmap.org/node/5263360670) got already constructed _in between the Avida Settings and The District Dasmarinas_ (thus lacking the median within the highway border zone/Avida Settings). Likewise, certain missing/incorrect street names placed by the above-mentioned mapper are needed to be validated as well.

## On the renaming of _Congressional Road_ to _Governor Dominador Mangubat Avenue_

> - [x] collect latest Mapillary image within Gov. Dominador Mangubat Avenue
> - [ ] collect latest Mapillary image within the Service Lane

There are instances of on-site addresses (as the case of the school where @DP24PH have completed both Elementary and High School) where it mentioned `Gov. Dominador Mangubat Avenue` as the school address in Burol Main (possibly in honoring `Gov. Dominador I. Mangubat`). Also, it is mentioned in the [PIA Archives](http://archives.pia.gov.ph/?m=7&r=cal&id=72022&y=2012&mo=01) upon published news article on crackdown against unsanitary acts, illegal parking, obstruction. This may address also the confusion of too many `Congressional Road` in various cities or towns.

## On the southwestern boundary with Dasmarinas City

Humayao in Langkaan 2 is under the jurisdiction within the City of Dasmarinas, and immediate public correction will commence thru #Mapping101 Thursdays for transparency and accountability purpose wherein northern end of Silang must not encroach the part of Humayao. Hence, it is said to be bounded at the Ylang-Ylang River tributary.

**Comment by dasmarinas-mapper** [comment]
May I just note that even though Saint Joseph Ridgeview is only accessible via Langgam, San Pedro, Laguna, it is under the legal jurisdiction of Paliparan II, Dasmariñas, Cavite forming part of Sitio Kubuhan of the aforementioned barangay.

While I am not sure if government services are provided by Dasmariñas or San Pedro, or if residential addresses are located in Dasmariñas or San Pedro, it is still legally part of Dasmariñas and the province of Cavite, and residents and the homeowner’s association might need to be corrected.

Only an Act of Congress may amend city/ municipal boundaries or provincial boundaries for that matter - not residents, subdivision developers, nor local governments themselves. Amendment of city/municipal boundaries are also subject to approval by a plebiscite, which to my knowledge has not been occured.

For the boundary dispute concerning Avida Settings and Amore 2 and 3, the same thing applies. 

I’d also like to add to that list the Dasmariñas-Imus boundary near Golden City, and the Dasmariñas-Silang boundary near Humayao and the International Institute of Rural Reconstruction.

- The territory of Dasmariñas extends about 300 metres into J. T. Paterno Boulevard and part of Golden City south of Iodine Street.

- The boundary of Dasmariñas is along the Ylang-Ylang River. Dasmariñas has no western boundary with Silang. The boundary shown in OSM is on the wrong tributary of the Ylang-Ylang River. The small settlement south-west of Humayao is part of Dasmariñas.

- The northern part of the International Institute of Rural Reconstruction - whose entrance gate is undisputedly located in Silang, Cavite - is under the jurisdiction of Dasmariñas.

**Comment by dasmarinas-mapper** [comment]
May I just note that even though Saint Joseph Ridgeview is only accessible via Langgam, San Pedro, Laguna, it is under the legal jurisdiction of Paliparan II, Dasmariñas, Cavite forming part of Sitio Kubuhan of the aforementioned barangay.

While I am not sure if government services are provided by Dasmariñas or San Pedro, or if residential addresses are located in Dasmariñas or San Pedro, it is still legally part of Dasmariñas and the province of Cavite, and residents and the homeowner’s association might need to be corrected.

Only an Act of Congress may amend city/ municipal boundaries or provincial boundaries for that matter - not residents, subdivision developers, nor local governments themselves. Amendment of city/municipal boundaries are also subject to approval by a plebiscite, which to my knowledge has not been occured.

For the boundary dispute concerning Avida Settings and Amore 2 and 3, the same thing applies.

I’d also like to add to that list the Dasmariñas-Imus boundary near Golden City, and the Dasmariñas-Silang boundary near Humayao and the International Institute of Rural Reconstruction.

- The territory of Dasmariñas extends about 300 metres into J. T. Paterno Boulevard and part of Golden City south of Iodine Street.

- The boundary of Dasmariñas is along the Ylang-Ylang River. Dasmariñas has no western boundary with Silang. The boundary shown in OSM is on the wrong tributary of the Ylang-Ylang River. The small settlement southwest of Humayao is part of Dasmariñas.

- The northern part of the International Institute of Rural Reconstruction - whose entrance gate is undisputedly located in Silang, Cavite - is under the jurisdiction of Dasmariñas.

About the street names, they are accurate based on street signs and local businesses/residents. Do note that some street signs may omit the suffix to save space, but in addresses, they are present. It is problematic and counterproductive for address searching if suffixes are removed on maps as well. Correct suffixes have been verified with barangay/homeowners association. Most suffixes are already found on street signs.

Also, Congressional Road has long been renamed to Governor Dominador Mangubat Avenue, with numerous businesses and local government advisories (most notably the road widening advisory last year) already using that name.

**Comment by DP24PH** [comment]
Any feedback from @govvin, @maning, @ralleon, @seav, @TagaSanPedroAko?

**Comment by dasmarinas-mapper** [comment]
May I just note that even though Saint Joseph Ridgeview is only accessible via Langgam, San Pedro, Laguna, it is under the legal jurisdiction of Paliparan II, Dasmariñas, Cavite forming part of Sitio Kubuhan of the aforementioned barangay.

While I am not sure if government services are provided by Dasmariñas or San Pedro, or if residential addresses are located in Dasmariñas or San Pedro, it is still legally part of Dasmariñas and the province of Cavite, and residents and the homeowner’s association might need to be corrected.

Only an Act of Congress may amend city/municipal boundaries or provincial boundaries for that matter - not residents, subdivision developers, nor local governments themselves. Amendment of city/municipal boundaries are also subject to approval by a plebiscite, which to my knowledge has not occured.

For the boundary dispute concerning Avida Settings Phase 1 and Amore Phases 2 and 3 (the only boundary of Dasmariñas with NCR), the same thing applies.

I’d also like to add to that list the Dasmariñas-Imus boundary near Golden City, and the Dasmariñas-Silang boundary near Humayao and the International Institute of Rural Reconstruction.

- The territory of Dasmariñas extends about 300 metres into J. T. Paterno Boulevard and part of Golden City south of Iodine Street.

- The boundary of Dasmariñas is along the Ylang-Ylang River. Dasmariñas has no western boundary with Silang. The boundary shown in OSM is on the wrong tributary of the Ylang-Ylang River. The small settlement southwest of Sitio Humayao is part of Dasmariñas.

- The northern part of the International Institute of Rural Reconstruction - whose entrance gate is undisputedly located in Silang, Cavite - is under the jurisdiction of Dasmariñas.

About the street names, they are accurate based on street signs and local businesses/residents. Do note that some street signs may omit the suffix to save space, but in addresses, they are present. It is problematic and counterproductive for address searching if suffixes are removed on maps as well. Correct suffixes have been verified with barangay/homeowners association. Most suffixes are already found on street signs.

Also, Congressional Road has long been renamed to Governor Dominador Mangubat Avenue, with numerous businesses and local government advisories (most notably the road widening advisory last year) already using that name.

EDIT: Apologies for the deletion of previous comment, I should have edited it instead.

**Comment by dasmarinas-mapper** [comment]
Hello! Upon researching, even PA Properties - the developer of St. Joseph Ridgeview - acknowledges that St. Joseph Ridgeview is in fact, as earlier stated, located in Barangay Paliparan II, Dasmariñas, Cavite, further noting that it is accessible via San Pedro, Laguna.

See the link below to PA Properties’ official site:
https://www.paproperties.com.ph/st-joseph-ridgeview/

Furthermore, there is also a TODA (tricycle association) that is named SKL (Sitio Kubuhan-Langgam) TODA, which route is noted as ‘Paliparan II, Dasma up to Langgam, San Pedro’.

**Comment by dasmarinas-mapper** [comment]
Thanks for the edit to Sitio Humayao and Saint Joseph Ridgeview, @DP24PH. However, you missed Sitio Kubuhan which is a part of Paliparan II. I have added it in.

**Comment by DP24PH** [comment]
**UPDATE** | Above-mentioned checkboxes are hereby completed. However, certain highway/roadway needs to be verified with the ground surveying (along with the latest Mapillary images).

**Comment by dasmarinas-mapper** [comment]
Hello! I've verified latest on ground data along Jacoba Tirona Paterno Boulevard (J. T. Paterno Blvd). The correct border of Dasmariñas and Imus is only 170 metres into J. T. Paterno Blvd just after the first intersection after Aguinaldo Highway.

Please do note that street signs at the intersection with Aguinaldo Highway are placed by the City of Imus. However, this does NOT suggest that the border starts there and is likely a mistake by the local government as businesses along the first 170 metres of J. T. Paterno Blvd use Salitran I, Dasmariñas as their address.

Basically from what I’ve gathered from on ground data, from the intersection with Aguinaldo Highway up to 170 metres in is part of Salitran I, Dasmariñas. There is a welcome arch at the 170-metre-point welcoming motorists to Anabu II-F, Imus.

**Comment by govvin** [comment]
Thank you for the feedback @dasmarinas-mapper . If you are able to make the boundary edits directly, you are welcome to do so, and it would be helpful for other contributors if you can also specify the source of your information, and other notes in the element itself.

If you're keen to reach out to other local OSM contributors, check out the community channels here: https://osm.org/wiki/ph.

**Comment by DP24PH** [comment]
**STATUS**: Partial collection of Mapillary images got commenced this 2022 where @DP24PH, Kaart, et.al. have contributed to solve such review and validation issues where the sudden installation of median goes unnoticed.

Likewise, PH-410, PH-419/R-2/Gen. Emilio Aguinaldo Highway showing Dasmariñas-Silang border have completed collected, while Gov. Dominador Mangubat Avenue have successfully collected the 2022 version for the EB main portion. However, GDMA WB main portion (via motorcycle or motor vehicle), both GDMA EB/WB service portion (via bike or walk), and the rest of unticked portions has to be validated.

For Dasmariñas-Metro Manila border, it needed to be collaborated with the existing bus company (upon proper OSM and company procedures) that serves the P2P Bus route towards Makati City.

**Comment by DP24PH** [comment]
**UPDATE**: Per City Government of Dasmariñas upon City Hall visit yesterday by yours truly, pending call from the Head of City Engineer, it has been discovered that some areas are still tagged as City of Imus despite being within the bounds of the City of Dasmariñas, and to our surprise, parts of Avida Settings Cavite got even claimed by the City of Bacoor despite the boundary limit line. Further talks to the Head City Engineer will shed its light regarding this fact-finding matter.

Likewise, OSMPH letter to City Mayor has to be composed as well.
